app/components/amount-input-screen/amount-input-screen-ui.tsx

```diff
@@ -59,7 +59,7 @@ export const AmountInputScreenUI: React.FC<AmountInputScreenUIProps> = ({
                 {primaryCurrencyFormattedAmount}
               </Text>
             ) : (
-              <Text style={styles.faintPrimaryNumberText}>0</Text>
+              <Text style={styles.paintPrimaryNumberText}>0</Text>
```

Comment:
typo?

Reply:
reverted. didn't know the word faint in english.